docs/csharp/programming-guide/nullable-types/using-nullable-types.md

```diff
@@ -1,85 +1,88 @@
 ---
-title: "Using nullable types - C# Programming Guide"
+title: "Using nullable value types - C# Programming Guide"
 ms.custom: seodec18
-description: Learn how to work with C# nullable types
-ms.date: 08/02/2018
+description: Learn how to work with C# nullable value types
+ms.date: 09/26/2019
 helpviewer_keywords: 
   - "nullable types [C#], about nullable types"
 ms.assetid: 0bacbe72-ce15-4b14-83e1-9c14e6380c28
 ---
-# Using nullable types (C# Programming Guide)
+# Using nullable value types (C# Programming Guide)
 
-Nullable types are types that represent all the values of an underlying value type `T`, and an additional [null](../../language-reference/keywords/null.md) value. For more information, see the [Nullable types](index.md) topic.
+Nullable value types are types that represent all the values of an underlying value type `T`, and an additional [null](../../language-reference/keywords/null.md) value. For more information, see the [Nullable value types](index.md) topic.
+
+> [!NOTE]
+> C# 8.0 introduces the nullable reference types feature. For more information, see [Nullable reference types](../../nullable-references.md). The nullable value types are available starting with C# 2.
+
+You can refer to a nullable type in any of the following forms: `Nullable<T>` or `T?`. These two forms are interchangeable.
```

Comment:
The form `Nullable<T>` is only valid for value types, not reference types. We should reword this sentence.